core/roboconf-dm-templating/metadata.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ipojo 
+		xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		xsi:schemaLocation="org.apache.felix.ipojo http://felix.apache.org/ipojo/schemas/CURRENT/core.xsd"
+		xmlns="org.apache.felix.ipojo">
+
+	<!-- A component for the DM's templating -->
+	<component classname="net.roboconf.dm.templating.internal.TemplatingManager" name="roboconf-dm-templating" public="false">
```

Comment:
What I like about this file is that we have iPojo information in a single place.
No need to introspect Java classes or search for annotations. Finding such information quickly may help to update [this page](http://roboconf.net/en/developer-guide/ipojo-model.html).